src/controllers/exercises/exercises.routes.js

```diff
@@ -0,0 +1,37 @@
+const router = require('express').Router();
```

Comment:
I like the chaining the `Router()` at the end, much cleaner than making then importing express as it's own variable, then making a router variable(which is what I've always done and seen people use).